app/assets/javascripts/hyrax/reports-buttons.js

```diff
@@ -0,0 +1,33 @@
+Blacklight.onLoad(function() {
+  var summaryButton = $('#summary-btn');
+  var monthlyButton = $('#monthly-btn');
+  var rangeButton = $('#range-btn');
+
+  $('#monthly-btn').click(function() {  
```

Comment:
I'd recommend switching this to `$('#monthly-btn').on('click', function() {` so any dynamically added elements get the event listener too.

Reply:
updated.  thank you!